code/parse_test.go

```diff
@@ -397,3 +397,61 @@ func TestElbGzipLogs(t *testing.T) {
+		},
+	}
+
+	logs := parseCloudWatchLogs(cloudWatchEvent)
+
+	time := time.Unix(0, 1619415915228*1000000)
```

Comment:
Can you convert from string to unix_time so that it's more readable.

Reply:
Implemented.

Reply:
It is better, How about we do it from string
`
t, err := time.Parse(time.RFC3339, "2014-11-12T11:45:26.371Z")
`

Reply:
Actually the test case are failing for above input also for the time := time.Date(2021, time.April, 27, 14, 25, 50, 324000000, loc). The timestamp that is getting calculated is different but it is running fine at my system. I think this might be location issue